### PR TITLE
HTTP-COOKIES.md: describe the cookie file format

### LINKED ISSUE #4805: document cookie format in CURLINFO_COOKIELIST
It would be nice if the CURLINFO_COOKIELIST had some documentation as to the format of the cookie strings that it returns, or a pointer to such documentation. (is it a standard format?)

By experimentation, I've determined that it seems to be tab-delimited, with a predictable ordering of the fields.

### PR DESCRIPTION
**Pull request by bagder**
... and refer to that file from from CURLOPT_COOKIEFILE.3 and
CURLOPT_COOKIELIST.3

Reported-by: bsammon on github
Fixes #4805

### CODE REVIEW

File: docs/HTTP-COOKIES.md [hunk 1]
```diff
@@ -43,6 +43,24 @@
...
+
+  Each line that each specifies a single cookie consists of seven text fields
+  separated with TAB characters.
+
+  | Field | Example     | Meaning                                       |
+  |-------|-------------|-----------------------------------------------|
+  | 0     | example.com | Domain name                                   |
+  | 1     | FALSE       | Include subdomains boolean                    |
+  | 2     | /foobar/    | Path                                          |
+  | 3     | TRUE        | Set over a secure transport                   |
```
Comment (jay) [inline docs/HTTP-COOKIES.md]: I would rephrase this to make it clear that this field indicates whether the cookie is HTTPS only.

Comment (bagder) [reply docs/HTTP-COOKIES.md]: Good point!

**Comment by jay** [comment]
Nitpick but the clarification I was looking for is set and sent over HTTPS _only_. IMHO "set over https" does not make that clear.

**Comment by bagder** [comment]
Thinking about some more, your explanation seem to focus more on when this file is *read* while my version when it is *written*. I think maybe it might need a third version where we think of both cases better...

**Comment by jay** [comment]
Set/sent over HTTPS only?
Maybe there should be a type column before the example column,  boolean, string, number. That could make it clearer.

**Comment by bagder** [comment]
Agreed. This looks much better now!

**Comment by bagder** [comment]
Thanks @jay!
